ldap/servers/slapd/main.c

```diff
@@ -2931,6 +2931,9 @@ slapd_do_all_nss_ssl_init(int slapd_exemode, int importexport_encrypt, int s_por
      * is enabled or not. We use NSS for random number generation and
      * other things even if we are not going to accept SSL connections.
      * We also need NSS for attribute encryption/decryption on import and export.
+     *
+     * It's important to remember that while in FIPS mode the administrator should always enable
```

Comment:
IMHO we should log an error if FIPS mode is enabled but not the security

Reply:
+1